tests/unit_tests/test_unit.py

```
@@ -456,13 +458,12 @@ def __call__(self, url):
         return response
 
 
-def test_setup_model(monkeypatch):
+def test_setup_model(monkeypatch, model_arg, expected_filename):
```

Ruff (0.15.14)

[warning] Too many statements (66 > 50)

(PLR0915)